<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coin</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../usr/share/icons/coin_48.png</normaloff>../../../../usr/share/icons/coin_48.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>580</y>
      <width>781</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>1001</width>
      <height>57</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,8,2,1,6" columnminimumwidth="0,0,0,0,0">
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEditAmount"/>
     </item>
     <item row="1" column="0">
      <widget class="QDateEdit" name="dateEdit">
       <property name="maximumSize">
        <size>
         <width>16777214</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="displayFormat">
        <string>M/d/yyyy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditTransactionInfo"/>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnAccept">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>29</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblDate">
       <property name="text">
        <string>date</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblTransactionInfo">
       <property name="text">
        <string>transaction info</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lblAmount">
       <property name="text">
        <string>amount</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QComboBox" name="comboAccounts"/>
     </item>
     <item row="0" column="4">
      <widget class="QCheckBox" name="transferCheckBox">
       <property name="text">
        <string>Transfer</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>btnAccept</zorder>
    <zorder>lineEditAmount</zorder>
    <zorder>lineEditTransactionInfo</zorder>
    <zorder>lblDate</zorder>
    <zorder>lblTransactionInfo</zorder>
    <zorder>lblAmount</zorder>
    <zorder>dateEdit</zorder>
    <zorder>comboAccounts</zorder>
    <zorder>transferCheckBox</zorder>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1001</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,3">
     <item row="1" column="0">
      <widget class="QTreeWidget" name="treeAccounts">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::NoDragDrop</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblAccounts">
         <property name="text">
          <string>accounts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddAccount">
         <property name="maximumSize">
          <size>
           <width>29</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDeleteAccount">
         <property name="maximumSize">
          <size>
           <width>29</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lblTransactions">
         <property name="text">
          <string>transactions</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditFilter"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>report</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QTableView" name="tableTransactions">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::NoDragDrop</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>22</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1019</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>dateEdit</tabstop>
  <tabstop>lineEditTransactionInfo</tabstop>
  <tabstop>lineEditAmount</tabstop>
  <tabstop>btnAccept</tabstop>
  <tabstop>treeAccounts</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditAmount</sender>
   <signal>returnPressed()</signal>
   <receiver>btnAccept</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>680</y>
    </hint>
    <hint type="destinationlabel">
     <x>554</x>
     <y>680</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditTransactionInfo</sender>
   <signal>returnPressed()</signal>
   <receiver>lineEditAmount</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>667</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>667</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
